### ISSUE TITLE
Multiple fields filtering small issue

### ISSUE THREAD

**Issue by saxamaphone69** [TuxedoTako/4chan-xt#29]
When using the General tab to filter, you can apply multiple types of filters. On 4chan X:
```
Filters in the "General" section apply to multiple fields, by default subject,name,filename,comment.
The fields can be specified with the type option, separated by commas.
For example: type:postID,name,uniqueID,tripcode,capcode,pass,email,subject,comment,flag,filename,dimensions,filesize,MD5;.
Types can also be combined with a + sign; this indicates the filter applies to the given fields joined by newlines.
For example: type:filename+filesize+dimensions;.
```
however on XT:
```
Filters in the "General" section apply to multiple fields, by default subject,name,filename,comment.
The fields can be specified with the type option, separated by commas.
For example: type:@{filterTypes};.
Types can also be combined with a + sign; this indicates the filter applies to the given fields joined by newlines.
For example: type:filename+filesize+dimensions;.
```

With it saying `type:@{filterTypes}` instead. Not idea if it even works as I don't use this type of filter.

**Comment by TuxedoTako** [comment]
`@{filterTypes}` is a placeholder used in the 4chan X build system. I missed that one when setting up the Rollup build XT uses. The filter syntax should still be the same. This will be fixed in the next release.

**Comment by TuxedoTako** [comment]
Should be fixed in [2.4.5](https://github.com/TuxedoTako/4chan-xt/releases).
